rosbag2_storage_mcap/src/mcap_storage.cpp

```diff
@@ -163,7 +248,13 @@ void MCAPStorage::open(const std::string& uri,
       relative_path_ = uri + FILE_EXTENSION;
 
       mcap_writer_ = std::make_unique<mcap::McapWriter>();
-      mcap::McapWriterOptions options{"ros2"};
+      McapWriterOptions options;
+      if (!storage_config_uri.empty()) {
+        YAML::Node yaml_node = YAML::LoadFile(storage_config_uri);
+        options = yaml_node.as<McapWriterOptions>();
+      }
+      options.profile = "ros2";
```

Comment:
This line makes me want to remove `profile` from the set of configurable options in `decode(const Node& node, McapWriterOptions& o)`